Core/src/org/sleuthkit/autopsy/core/UserPreferences.java

```diff
@@ -45,6 +46,7 @@
     public static final String HIDE_SLACK_FILES_IN_DATA_SRCS_TREE = "HideSlackFilesInDataSourcesTree"; //NON-NLS 
     public static final String HIDE_SLACK_FILES_IN_VIEWS_TREE = "HideSlackFilesInViewsTree"; //NON-NLS 
     public static final String DISPLAY_TIMES_IN_LOCAL_TIME = "DisplayTimesInLocalTime"; //NON-NLS
+    public static final String CUSTOM_TIME_ZONE = "CustomTimeZone"; //NON-NLS
```

Comment:
@dgrove727 and @wschaeferB, does this need to be public? I realize that the precedent is mixed, but it looks to me like the previous practice of exposing of the names for name-value pairs in the underlying NbPreferences object was not necessary nor desirable. Am I missing something?

Reply:
This is used by DirectoryTreeTopComponent to refresh whenever the time zone option is changed.